src/main/scala/mesosphere/marathon/raml/PodStatusConversion.scala

```diff
@@ -249,36 +247,39 @@ trait PodStatusConversion {
     }.getOrElse(List.empty[ContainerEndpointStatus])
 
   def podInstanceState(
-    instanceCondition: core.condition.Condition,
+    instance: core.instance.Instance,
```

Comment:
What is the motivation for this change?

Reply:
I needed to do that because of this line https://github.com/mesosphere/marathon/pull/6658/files#diff-e0df4221bad191a89f0d04c9d30bbd06R253

Reply:
Got it! Thanks!